### PR TITLE
Adding ability to equip hotkeys.

### PR DESCRIPTION
**Pull request by slavidodo**
Thanks to @ninjalulz .

### CODE REVIEW

File: src/creature.h [hunk 1]
```diff
@@ -48,6 +48,16 @@ enum slots_t : uint8_t {
 	CONST_SLOT_LAST = CONST_SLOT_AMMO,
 };
 
++enum clientSlots_t : uint16_t {
```
Comment (esfomeado) [inline src/creature.h]: Extra '+' here

Comment (slavidodo) [inline src/creature.h]: Edited, Thanks.

File: src/protocolgame.cpp [hunk 2]
```diff
@@ -818,6 +819,34 @@ void ProtocolGame::parseUpdateContainer(NetworkMessage& msg)
 	addGameTask(&Game::playerUpdateContainer, player->getID(), cid);
 }
 
+void ProtocolGame::parseEquipObject(NetworkMessage& msg) {
```
Comment (esfomeado) [inline src/protocolgame.cpp]: Should be parseEquipHotkey

File: src/protocolgame.cpp [hunk 3]
```diff
@@ -432,6 +432,7 @@ void ProtocolGame::parsePacket(NetworkMessage& msg)
 		case 0x70: addGameTaskTimed(DISPATCHER_TASK_EXPIRATION, &Game::playerTurn, player->getID(), DIRECTION_EAST); break;
 		case 0x71: addGameTaskTimed(DISPATCHER_TASK_EXPIRATION, &Game::playerTurn, player->getID(), DIRECTION_SOUTH); break;
 		case 0x72: addGameTaskTimed(DISPATCHER_TASK_EXPIRATION, &Game::playerTurn, player->getID(), DIRECTION_WEST); break;
+		case 0x77: parseEquipObject(msg); break;
```
Comment (esfomeado) [inline src/protocolgame.cpp]: Should be parseEquipHotkey

**Comment by marksamman** [comment]
There are a couple of problems with this:
1) you can't just copy someone elses code and submit it as a pull request, you need their permission as the code becomes licensed under GPL when we merge it
2) your commit messages do not describe what you have changed
3) this only works for items you had equipped when you logged in, and sending every equipped item every time something in the inventory changes is too costly

**Comment by slavidodo** [comment]
Understood, Close it please.

**Comment by ranisalt** [comment]
Whose code is this? It's actually pretty neat, but the interval between sending inventory changes could be a debounced interval of, say, 5 seconds, meaning 5 seconds after the last change, if no changes occur, we send an update.

**Pull request closed without merging**